network-api/networkapi/wagtailpages/templates/wagtailpages/static/participate_page2.html

```diff
@@ -42,8 +42,9 @@ <h2 class="tw-h2-heading">{{ page.h2 }}</h2>
 
         <div class="my-5">
             <div class="row">
-                {% for entry in page.featured_highlights2.all %}
+                {% for entry in page.ordered_featured_highlights2 %}
                     {% with highlight=entry.highlight %}
+                        <p>{{ highlight.title }}</p>
```

Comment:
was this for testing purposes? :)

Reply:
Oopsie :see_no_evil: removed now, thanks!